tests/subsys/module_hello/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  subsys.module_hello:
+    tags: extensibility
```

Comment:
I'd like to use a tag for easily narrowing testing to those needed for verifying extensibility via `module.yml`.  What tags: should be used here?